libcaf_net/caf/detail/rfc6455.hpp

```diff
@@ -17,49 +17,57 @@ namespace caf::detail {
 struct CAF_NET_EXPORT rfc6455 {
   // -- member types -----------------------------------------------------------
 
+  enum class opcode_type : uint8_t {
+    continuation_frame = 0x00,
+    text_frame = 0x01,
+    binary_frame = 0x02,
+    connection_close = 0x08,
+    ping = 0x09,
+    pong = 0x0A,
+    /// Invalid opcode to mean "no opcode received yet".
+    nil_code = 0xFF,
+  };
+
   struct header {
-    bool fin;
-    uint8_t opcode;
-    uint32_t mask_key;
-    uint64_t payload_len;
+    bool fin = false;
+    opcode_type opcode = opcode_type::nil_code;
+    uint32_t mask_key = 0;
+    uint64_t payload_len = 0;
```

Comment:
@Neverlord basically I only need the `opcode` initialized here. Does it make sense if I initialize all fields, or should I, for performance reasons, only init the `opcode`?